sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -239,10 +239,14 @@ public abstract partial class Response : System.ClientModel.Primitives.PipelineR
     {
         protected Response() { }
         public abstract string ClientRequestId { get; set; }
+        public virtual new System.BinaryData Content { get { throw null; } }
```

Comment:
We require this to be newslotted if we want the base type in ClientModel to be non-virtual.